src/order/complete_lattice.lean

```diff
@@ -159,11 +161,17 @@ is_glb_singleton.Inf_eq
 
 end
 
-/-- A complete lattice is a bounded lattice which
+/-- A complete lattice is a bounded order which
   has suprema and infima for every subset. -/
-@[protect_proj]
+@[protect_proj, ancestor lattice complete_semilattice_Sup complete_semilattice_Inf has_top has_bot]
```

Comment:
Does anything actually use these? I thought this was only for to_additive. At worst it's harmless.